backend/backend.go

```diff
@@ -592,12 +626,8 @@ func (backend *Backend) ManualReconnect(reconnectETH bool) {
 	}
 	if reconnectETH {
 		backend.log.Info("Reconnecting ETH accounts")
-		for _, account := range backend.accounts {
-			ethAccount, ok := account.(*eth.Account)
-			if !ok {
-				continue
-			}
-			ethAccount.EnqueueUpdate()
+		if err := backend.updateETHAccounts(); err != nil {
```

Comment:
This makes a call in parallel with the existing loop and does not obey the poll interval, right? 

Since the backend has a poll loop, why is there even a poll loop left in account.go? Could move all the polling logic of account.go (including enqueing an update, quitting etc) there instead of keeping two separate poll loops.

Reply:
This is still open - you want to kick the pollETHAccounts() loop into action (with a channel) so the next call is scheduled for 5min after this one